src/libs/antares/checks/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+set(SRC_CHECKS
+        checkLoadedInputData.cpp
+        antares/checks/checkLoadedInputData.h
+)
+source_group("checks" FILES ${SRC_CHECKS})
+
+add_library(checks
+        ${SRC_CHECKS}
+)
+
+target_include_directories(checks
+        PRIVATE
+        ${CMAKE_SOURCE_DIR}/libs/
```

Comment:
Basically, we need all those include directories as soon as we include study.h, from what I understand.

This is actually a cyclic dependency on "core" lib. Because we don't make it explicit for cmake, we need to include "by hand" all the necessary include directories, instead of letting cmake do it.

I would propose to make it explicit since it's the current situation, by including libantares-core in target_link_libraries.

This will avoid all the include directories that are added after, and make explicit that cyclic dependency.

In the future, we should probably target to remove those cyclic dependencies.

Reply:
Probably for a future PR though ...